wallet/createtx_test.go

```diff
@@ -399,3 +402,87 @@ func TestCreateSimpleCustomChange(t *testing.T) {
 		require.Equal(t, scriptType, txscript.WitnessV0PubKeyHashTy)
 	}
 }
+
+// TestSelectUtxosTxoToOutpoint tests that it is possible to use passed
+// selected utxos to craft a transaction in `txToOutpoint`
+func TestSelectUtxosTxoToOutpoint(t *testing.T) {
+	t.Parallel()
+
+	w, cleanup := testWallet(t)
+	defer cleanup()
+
+	// First, we'll make a P2TR and a P2WKH address to send some coins to.
+	p2wkhAddr, err := w.CurrentAddress(0, waddrmgr.KeyScopeBIP0084)
+	require.NoError(t, err)
+
+	p2trAddr, err := w.CurrentAddress(0, waddrmgr.KeyScopeBIP0086)
+	require.NoError(t, err)
+
+	// We'll now make a transaction that'll send coins to both outputs,
+	// then "credit" the wallet for that send.
+	p2wkhScript, err := txscript.PayToAddrScript(p2wkhAddr)
+	require.NoError(t, err)
+
+	p2trScript, err := txscript.PayToAddrScript(p2trAddr)
+	require.NoError(t, err)
+
+	incomingTx := &wire.MsgTx{
+		TxIn: []*wire.TxIn{
+			{},
+		},
+		TxOut: []*wire.TxOut{
+			wire.NewTxOut(1_000_000, p2wkhScript),
+			wire.NewTxOut(2_000_000, p2trScript),
+			wire.NewTxOut(3_000_000, p2trScript),
+			wire.NewTxOut(7_000_000, p2trScript),
+		},
+	}
+	addUtxo(t, w, incomingTx)
+
+	// we expect 4 unspent utxos
```

Comment:
nit: incomplete sentence in comment, here and below (capitalization and punctuation).